<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Floor1</class>
 <widget class="QWidget" name="Floor1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tầng 1</string>
  </property>
  <widget class="QPushButton" name="pushButton_Light">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>291</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: 0px</string>
   </property>
   <property name="text">
    <string>Đèn</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/off-button.png</normaloff>
     <normalon>:/resources/on-button.png</normalon>:/resources/off-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>200</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_Humidity">
      <property name="text">
       <string>Độ ẩm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_Temp_Value">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Fire">
      <property name="text">
       <string>Nguy cơ cháy nổ</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_Humidity_Value">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Temp">
      <property name="text">
       <string>Nhiệt độ </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_Fire_Value">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Curtain">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: 0px</string>
   </property>
   <property name="text">
    <string>rèm cửa</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/off-button.png</normaloff>
     <normalon>:/resources/on-button.png</normalon>:/resources/off-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>200</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Home">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: 0px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/homepage.png</normaloff>:/resources/homepage.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Home">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Màn hình chính</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Light_Status">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>190</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/resources/light_on.png</normaloff>:/resources/light_on.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
